db/schema.rb

```diff
@@ -430,7 +430,6 @@
     t.datetime "opening_at"
     t.datetime "closing_at"
     t.integer  "corporate_information_page_type_id",          limit: 4
-    t.string   "need_ids",                                    limit: 255
```

Comment:
It looks like a migration has been run to remove this field but the migration isn't in this PR. I meant checkout the schema from master to put this field back and reset this change.